src/mbgl/annotation/shape_annotation_impl.cpp

```diff
@@ -88,21 +88,22 @@ void ShapeAnnotationImpl::updateTile(const CanonicalTileID& tileID, AnnotationTi
         const double tolerance = baseTolerance / (maxAmountOfTiles * util::EXTENT);
 
         geojsonvt::ProjectedRings rings;
-        std::vector<geojsonvt::LonLat> points;
```

Comment:
These changes will conflict with https://github.com/mapbox/mapbox-gl-native/compare/5158-minimal but the resolution should be straightforward.